controllers/apps/reconcile.go

```diff
@@ -122,21 +122,9 @@ func (handler *Handler) updateEmqxStatus(emqx v1beta3.Emqx) error {
 	emqx.DescConditionsByTime()
 	err := handler.client.Status().Update(context.TODO(), emqx)
 	if err != nil {
-		handler.logger.WithValues(
-			"kind", emqx.GetKind(),
-			"apiVersion", emqx.GetAPIVersion(),
-			"namespace", emqx.GetNamespace(),
-			"name", emqx.GetName(),
-			"conditions", emqx.GetConditions(),
-		).Error(err, "Update emqx broker status unsuccessfully")
+		handler.eventsCli.Event(emqx, corev1.EventTypeWarning, "Reconciled", err.Error())
 		return err
 	}
-	handler.logger.WithValues(
-		"kind", emqx.GetKind(),
-		"apiVersion", emqx.GetAPIVersion(),
-		"namespace", emqx.GetNamespace(),
-		"name", emqx.GetName(),
-		"conditions", emqx.GetConditions(),
-	).V(3).Info("Update emqx broker status successfully")
+	handler.eventsCli.Event(emqx, corev1.EventTypeNormal, "Reconciled", "Update emqx broker status successfully")
```

Comment:
```suggestion
	handler.eventsCli.Event(emqx, corev1.EventTypeNormal, "Reconciled", "Update status successfully")
```
Maybe delete “emqx broker”, because this event already in emqx custom resource

Reply:
Resolved